fda/basis.py

```diff
@@ -17,6 +17,8 @@
 from scipy.special import binom
 
 from fda import grid
+import fda.registration
```

Comment:
Please, use relative paths (`from . import grid`, `import . registration`). I know that existing ones are absolute, but I will address that in a future PR (I want to change the package name to `skfda`).

Reply:
Changed in 75b42f7.